### PR TITLE
Saved view tweaks

### PR DESCRIPTION
**Pull request by benjaminpkane**
Tweaks for embedded saved views

### CODE REVIEW

File: app/packages/dataset/src/Dataset.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ import styled from "styled-components";
 // built-in plugins
 import "@fiftyone/looker-3d";
 import "@fiftyone/map";
+import { useRouter } from "next/router";
```
Comment (kaixi-wang) [inline app/packages/dataset/src/Dataset.tsx:28]: Do we use the next router in oss?

Comment (benjaminpkane) [reply app/packages/dataset/src/Dataset.tsx:28]: No, I'll move

File: app/packages/dataset/src/Dataset.tsx [hunk 2]
```diff
@@ -57,6 +56,7 @@ export interface DatasetProps {
   theme?: "dark" | "light";
   toggleHeaders?: () => void;
   canEditSavedViews?: boolean;
+  savedViewSlug: string;
```
Comment (kaixi-wang) [inline app/packages/dataset/src/Dataset.tsx:59]: should we make this optional?

Comment (benjaminpkane) [reply app/packages/dataset/src/Dataset.tsx:59]: good point

**Review by kaixi-wang — APPROVED** [review]
lgtm

**Pull request merged**